src/bridge/MatrixHandler.ts

```diff
@@ -1132,7 +1132,7 @@ export class MatrixHandler {
                 const explanation = renderTemplate(this.config.truncatedMessageTemplate, { url: httpUrl });
                 let messagePreview = trimString(
                     potentialMessages[0],
-                    ircClient.getMaxLineLength() - 4 /* "... " */ - explanation.length
+                    ircClient.getMaxLineLength() - 4 /* "... " */ - explanation.length - ircRoom.channel.length
```

Comment:
This looks fine, are we sure this is the only place it is needed?

Reply:
It's the only place where we use `getMaxLineLength()` so it's probably the only wrong calculation thereof :)